research/index.md

```diff
@@ -1,77 +1,19 @@
-               </a>
-               <div class="stream-meta article-metadata">
-                 <div><span><a href="/author/nelson-bighetti/">Nelson Bighetti</a></span><span title="Equal contribution" class=author-notes>(?)</span>, <span>Robert Ford</span><span title="Equal contribution" class=author-notes>(?)</span></div>
-               </div>
-               <div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://arxiv.org/pdf/1512.04133v1" target=_blank rel=noopener>PDF</a>
-                 <button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/journal-article/cite.bib>
-                   Cite</button>
-                   <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/slides/example/" target=_blank>Slides</a></div>
-                 </div>
-                 <div class=ml-3><a href="/publication/journal-article/"><img src="/assets/img/paper2.jpg" alt="An example journal article"></a>
-                 </div>
-               </div>
-               <div class="media stream-item">
-                 <div class=media-body>
-                   <h3 class="article-title mb-0 mt-0"><a href="/publication/conference-paper/">An example conference paper</a>
-                   </h3>
-                   <a href="/publication/conference-paper/" class=summary-link>
-                     <div class=article-style>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.</div>
-                   </a>
-                   <div class="stream-meta article-metadata">
-                     <div><span><a href="/author/nelson-bighetti/">Nelson Bighetti</a></span>, <span>Robert Ford</span></div>
-                   </div>
-                   <div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://eprints.soton.ac.uk/352095/1/Cushen-IMV2013.pdf" target=_blank rel=noopener>PDF</a>
-                      <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/zalandoresearch/fashion-mnist" target=_blank rel=noopener>Code</a>
-                       <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/#" target=_blank rel=noopener>Dataset</a>
-                       <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://example.org" target=_blank rel=noopener>Custom Link</a></div>
-                     </div>
-                     <div class=ml-3><a href="/publication/conference-paper/"><img src="/assets/img/paper3.jpg" alt="An example conference paper"></a>
-                     </div>
-                   </div>
-                 </div>
-               </div>
-             </div>
-           </section>
-~~~
+{{allitems research/talks.toml}}
+
+\end{itemslist}
+
+\begin{itemslist}{name="Posters", class=""}
```

Comment:
Might be nice to have a function that takes care of putting in the itemslist as well to avoid this boilerplate?